src/main/scala/scalismo/common/interpolation/BarycentricInterpolator.scala

```diff
@@ -29,17 +30,38 @@ case class BarycentricInterpolator3D[A: ValueInterpolator](m: TetrahedralMesh[_3
 
   override protected val valueInterpolator: ValueInterpolator[A] = ValueInterpolator[A]
 
-  private def getTetrahedralMeshCell(p: Point[_3D]): TetrahedralCell = {
-    val closestPoint = m.pointSet.findClosestPoint(p)
-    val adjacentTetrahedra = m.tetrahedralization.adjacentTetrahedronsForPoint(closestPoint.id)
-    val tetraId = adjacentTetrahedra.filter(tId => m.isInsideTetrahedralCell(p, m.tetrahedralization.tetrahedrons(tId.id))).head
-    m.tetrahedralization.tetrahedrons(tetraId.id)
+  // Temporary solution, replace for Milestone M2!
```

Comment:
please use FIXME or TODO here, as both eclipse and intellij will then show the comment in the to do tab.